webpack/JobInvocationDetail/TemplateInvocationComponents/OutputCodeBlock.js

```diff
@@ -55,6 +55,47 @@ export const OutputCodeBlock = ({ code, showOutputType, scrollElement }) => {
   const filteredCode = code.filter(
     ({ output_type: outputType }) => showOutputType[outputType]
   );
+
+  const scrollElementSeleceted = useCallback(
+    () => document.querySelector(scrollElement),
+    [scrollElement]
+  );
+  const onClickScrollToTop = useCallback(() => {
+    scrollElementSeleceted().scrollTo(0, 0);
+  }, [scrollElementSeleceted]);
+
+  const onClickScrollToBottom = useCallback(() => {
+    scrollElementSeleceted().scrollTo(0, scrollElementSeleceted().scrollHeight);
+  }, [scrollElementSeleceted]);
+
+  const [isScrolledBottom, setIsScrolledBottom] = useState();
+  useEffect(() => {
```

Comment:
We cant check "isScrolledBottom" after the new output has been received already, so we need to keep checking if the user scrolled to the bottom